src/ompl/geometric/planners/informedtrees/BITstar.h

```diff
@@ -149,7 +149,7 @@ namespace ompl
             using NameFunc = std::function<std::string()>;
 
             /** \brief Construct with a pointer to the space information and an optional name. */
-            BITstar(const base::SpaceInformationPtr &spaceInfo, const std::string &name = "BITstar");
+            BITstar(const base::SpaceInformationPtr &spaceInfo, const std::string &name = "kBITstar");
```

Comment:
was this intentional?

Reply:
Yes. Since c08da89 the k-nearest connectivity model is the default in BIT*, but in BIT*'s `setup` method we have the following check, which produces a warning and changes the name to `"kBITstar"` if the k-nearest model is active.
https://github.com/ompl/ompl/blob/25c76431cef25f0100ed74d09dd88944ecca5ee1/src/ompl/geometric/planners/informedtrees/src/BITstar.cpp#L240-L246
I don't know if the above check is really necessary, but I don't think its great to have the default settings result in a warning.